src/mono/mono/mini/mini-ops.h

```diff
@@ -1021,6 +1021,7 @@ MINI_OP(OP_SSE_AND, "sse_and", XREG, XREG, XREG)
 MINI_OP(OP_SSE_OR, "sse_or", XREG, XREG, XREG)
 MINI_OP(OP_SSE_XOR, "sse_xor", XREG, XREG, XREG)
 MINI_OP(OP_SSE_ANDN, "sse_andn", XREG, XREG, XREG)
+MINI_OP(OP_SSE_ADDS, "sse_adds", XREG, XREG, XREG)
```

Comment:
Minor nitpick: paddsw/paddsb are SSE2, not SSE.

Reply:
Renamed, in `mini-llvm.c` for `INTRINS_SSE_*` enum constants we don't use version suffix so I didn't change that